src/transformers/modeling_utils.py

```diff
@@ -1507,6 +1507,24 @@ def _tie_or_clone_weights(self, output_embeddings, input_embeddings):
         if hasattr(output_embeddings, "out_features") and hasattr(input_embeddings, "num_embeddings"):
             output_embeddings.out_features = input_embeddings.num_embeddings
 
+    def _get_no_split_modules(self, device_map):
```

Comment:
- For the accelerate tests, this will be tested once this [PR](https://github.com/huggingface/transformers/pull/27001/files) is merged (inside ModelTesterMixin).
- For the documentation, i will do it. 
-  If `self._split_module is not []`, we should still check the underlying modules. This will become confusing otherwise. Moreover, I don't think that iterating model.modules() takes a lot of time. 
- For the CI, I don't know if we should do it. This will slow down the addition of new models. cc @LysandreJik

Reply:
Not entirely sure we need to have it defined by default for new models, especially as some models are on the smaller side and don't need it

Reply:
OK! What I mean by a test on the ci is to make sure that new models have a no split modules to support accelerate no test the model! 😉 but it might already be there